### DISCUSSION TITLE
v1.2 for Scope 3 Category 15

### DISCUSSION THREAD

**Discussion by Brian-Solomon** [USEPA/supply-chain-factors#60] [Q&A]
We employed the "Industry" dataset from prior USEEIO versions to estimate emissions from Scope 3 Category 15 investments. However, the Industry dataset is no longer available in the v1.2 release. Does EPA have any suggestions on how to use the current v1.2 for Category 15 calculations? Thank you.

**Comment by bl-young** [comment]
Are you referring to the fact that the current release is a Commodity model and not an Industry model? Yes there is currently not a specific release that includes Industry factors, but those can be generated with the R code.

See https://github.com/USEPA/supply-chain-factors/discussions/42
